pandas/core/indexes/base.py

```diff
@@ -2182,7 +2182,7 @@ def _nan_idxs(self):
         if self._can_hold_na:
             return self._isnan.nonzero()[0]
         else:
-            return np.array([], dtype=np.int64)
+            return np.array([], dtype=np.intp)
```

Comment:
shouldn't this cause a test breakage (on 32-bit)? (or are we not testing this particular case)

Reply:
looks like we only really use _nan_idxs in `Float64Index.get_loc`, so this line wont be reached anyway